src/TesApi.Web/Startup.cs

```diff
@@ -75,7 +75,7 @@ public void ConfigureServices(IServiceCollection services)
 
                     .AddMemoryCache(o => o.ExpirationScanFrequency = TimeSpan.FromHours(12))
                     .AddSingleton<ICache<TesTaskDatabaseItem>, TesRepositoryCache<TesTaskDatabaseItem>>()
-                    .AddSingleton<TesTaskPostgreSqlRepository>()
+                    .AddSingleton(sp => ActivatorUtilities.CreateInstance<TesTaskPostgreSqlRepository>(sp))
```

Comment:
This change is redundant (the other syntax does exactly this, and it was easier to read)